packages/examples/packages/preinstalled/README.md

```diff
@@ -5,3 +5,26 @@ MetaMask extension by default.
 
 > [!NOTE]
 > Preinstalled Snaps are primarily for internal use by MetaMask.
+
+This snap also demonstrates how to use the `endowment:page-settings` permission to show a settings page that leverages custom UI components.
+
+> [!NOTE]
+> This endowment is initially restricted to preinstalled snaps only.
+
+## Snap manifest
+
+The manifest of this snap includes the `endowment:page-settings` permission:
```

Comment:
We should make a note that we likely want to have this as a separate example Snap later. E.g. once we give the capability to non preinstalled Snaps

Reply:
https://github.com/MetaMask/snaps/issues/2963